### ISSUE TITLE
best move arrows cause graphical glitch - iOS analysis mode

### ISSUE THREAD

**Issue by cosmic4z** [veloce/lichobile#1279]
I am using latest ver of the app on iPad mini 7 iOS 12.4.1. In analysis mode, and only if best move arrows are on, I get some graphical glitches (happens every time for me if arrows are on).

Looks like some artefact related to the drawing of the arrows, as these graphical glitches update and move at around the same frequency as the arrows.

**Comment by sangwal** [comment]
![Screenshot_20200716-185817](https://user-images.githubusercontent.com/32763625/87677616-81b77980-c769-11ea-9337-7cccfe77c6a6.png)

Found similar thing on Android also.

**Comment by RogerThiede** [comment]
I suspect @cosmic4z and @sangwal are describing different graphical glitches.  @cosmic4z, can you post pictures of your glitch?  Is it similar to ornicar/lila#7721?

**Comment by cosmic4z** [comment]
Yes (having taken a look at the screenshot) I very much suspect it’s the same issue as the one you linked (#7721). I believe it’s linked to the drawing code for the best move arrows. Happy to post screenshot if requested.

**Comment by Unclevinny** [comment]
Is this really the same issue as my bug, #7721? In my bug, there are black lines that draw horizontally/vertically. The screenshot above seems to be about the best move arrow drawing to a square that doesn't make sense. 

[Here are my screenshots again](https://photos.google.com/share/AF1QipNEgqlAt9Ucgyr7FyCWJy2ZDZJkT5w9J7j9yN28aU9SHe7AQDopF3eEJhPELqmrkw?key=V2JDaVRKVUhWUk1QNEhnU0g1cW9kaEZzU1oxUnFR), and I'll paste one in to save you a click. 

![Screen Shot 2020-12-28 at 2 19 34 PM](https://user-images.githubusercontent.com/879631/103246278-c437d180-4917-11eb-9af3-46df2eae5914.png)

**Comment by Unclevinny** [comment]
Hello, just checking in on this. The black lines (not the incorrect arrows as in @sangwal's screenshot) still show for me.

**Comment by mrudolph2** [comment]
Problem still exists. Occurs as soon as I set “multiple lines” to anything other than 1.

iPad Pro 11-inch 2022, iPadOS 16.1.1
